lib/icinga/checkable.ti

```diff
@@ -70,9 +70,7 @@ abstract class Checkable : CustomVarObject
 		}}}
 	};
 	[config] bool volatile;
-	[config] double flapping_threshold {
```

Comment:
We should add a note to the release notes that this config attribute was removed.

Reply:
Done with new commit